examples/nakama-cmake-client-example/build/android/build_android.py

```diff
@@ -19,8 +19,8 @@
 import os
 
 if len(sys.argv) < 2:
-    print "Pass ABI parameter."
-    print "e.g. armeabi-v7a, arm64-v8a or x86"
+    print("Pass ABI parameter.")
```

Comment:
It seems time to upgrade python to v3 :+1: 
Could you please send python related changes as separate PR?
Thank you for contributing!